File: Chapter09/Java/PreferencesDemo/app/src/main/java/com/androidbook/preferences/sample/FlightPreferenceActivity.java
package com.androidbook.preferences.sample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class FlightPreferenceActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.flight_preferences_activity);
        getSupportFragmentManager()
                .beginTransaction()
                .replace(R.id.settings_container, new FlightPreferenceFragment())
                .commit();
    }
}

File: Chapter09/Kotlin/PreferencesDemo/app/src/main/java/com/androidbook/preferences/sample/FlightPreferenceActivity.kt
package com.androidbook.preferences.sample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class FlightPreferenceActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_flight_preference)
        supportFragmentManager.beginTransaction()
            .replace(R.id.settings_container, FlightPreferenceFragment())
            .commit()
    }
}